src/EventListeners.ts

```diff
@@ -46,7 +46,7 @@ import { changeTab, changeSubTab } from './Tabs'
     Platonic and/or Khafra have the right to close PRs that do not conform to this style guide
 
     If you are editing this script, please update the below time:
-    Last Edited: June 10, 2021 3:04 AM UTC-8
+    Last Edited: Sunday, 28-May-23 22:36:58 UTC
```

Comment:
for future reference I don't know what Platonic was going for here but you don't need to update comments like these